numba/tests/test_warnings.py

```diff
@@ -4,9 +4,17 @@
 
 import numba.unittest_support as unittest
 from numba import jit
-from numba.errors import NumbaWarning, deprecated
+from numba.errors import NumbaWarning, deprecated, NumbaDeprecationWarning
 
 class TestBuiltins(unittest.TestCase):
+
+    def check_objmode_deprecation_warning(self, w):
+        # Object mode fallback is slated for deprecation, check the warning
+        msg = ("Fallback from the nopython compilation path to the object "
```

Comment:
`Fallback` spelling (without hyphen) is inconsistent with the exceptions and documentation using `fall-back` (with hyphen).  I don't know which is more correct, although I would have used "fallback" by default.  Regardless, the difference is causing the unittests to fail in CI.

Reply:
Gone with `fall-back` consistently for user facing feedback on the basis that `fallback` is not a word.